docs/pages/docs/faq.mdx

```diff
@@ -52,6 +52,174 @@ defined and incompatible interfaces - which are the biggest problems in common
 specifications.
 </FAQBox>
 
+<FAQBox title="How does Quint compare to TLA+?">
+
+Quint is based on TLA+ and uses the same underlying logic (Temporal Logic of
+Actions), so it is very similar to TLA+ in terms of where it is useful. The main
+differences between Quint and TLA+ are:
+1. Quint has programming-style syntax, while TLA+ uses math/latex symbols.
+2. Quint has more static analysis, such as type checking and different modes
+   (`pure def`, `action`, etc), while TLA+ has no native support for types nor
+   explicit distinction between the modes (or levels, as they are called in
+   TLA+'s documentation).
+3. Quint tooling is more similar to what we have for programming languages:
+   near-instant feedback as you type the code, CLI-first executable, an easy way
+   to obtain one execution of your spec in the terminal, a REPL, located error
+   messages, etc. These are all not currently available in TLA+, or available
+   with significant constraints.
+4. Quint won't let you write many things that are allowed in TLA+. TLA+ has a
+   proof system (TLAPS), while Quint doesn't. Therefore, it is possible to write
+   more advanced formulas in TLA+ than in Quint, even ones that are not
+   supported by the existing model checkers, and then prove them using the proof
+   system. Quint restricts you to a fragment of TLA with the intention of
+   preventing unintentional usage of operators that lead to complicated
+   behaviors that are hard to debug and understand. Our understanding is that
```

Comment:
Hmmm I don't think we can attribute state space management to the constraints of Quint. What I have in mind here is that in TLA+ you can write `x = x'` for `x' = x`, which is exactly the same in terms of state space, but more complex to understand if you are not thinking about math. Then, there are the weirder ones like `(x + 1)' =  x` and `f'(x')` etc, and the ones which would actually affect the search space as `x' > x`.

But, mostly, we are just trying to avoid people using the prime operator in different ways as it is not obvious what it would do, and people might expect something different from what it actually means. This also happens with non determinism.

My sentence comes from me believing that people who are more used to math than to programming will like TLA+ better than Quint (at least for the syntax, and they might even be not that bothered by the tooling). A good rule of thumb might be: would you rather read a map like `{ f(x) : x ∈ S }` or `S.map(x => f(x))`? I can read both fine, but I prefer the second one by far. But there are definitely people who prefer the first.